### PR TITLE
chore: updates polygon allowlist

### PR DESCRIPTION
**Pull request by nickbytes**
## What changed (plus any additional context for devs)
- Updates polygon allowlist for next release

## Screen recordings / screenshots
- Not applicable

## What to test
- Not applicable, feature is already tested.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Added e2e tests? If not, please specify why
  - Feature is already tested

### CODE REVIEW

File: src/references/polygon-allowlist.json [hunk 1]
```diff
@@ -93,5 +93,6 @@
     "0xcb06f6aee0655252a3f6f2884680421d55d3c645",
     "0x4cd15dcd96362cf85e19039c3c2d661e5e43145e",
     "0x0a2568fde974948cfa6541193d3e018d9451a932",
-    "0xd79cf5a41d8caec4688e01b4754ea2da6f51e856"
+    "0xd79cf5a41d8caec4688e01b4754ea2da6f51e856",
+    "0xAA3181bf533fa5ce8ae8De036EdCe2e7d43b8a96"
```
Comment (skylarbarrera) [inline src/references/polygon-allowlist.json:97]: lowercase

**Review by skylarbarrera — APPROVED** [review]

File: src/references/polygon-allowlist.json (hunk 1 above)
Comment (nickbytes) [reply src/references/polygon-allowlist.json]: Updated to lowercase

**Comment by nickbytes** [comment]
@skylarbarrera was Taylor saying we needed to add some more here for upcoming release?

**Review by Shudogg — COMMENTED** [review]
Good

**Pull request merged**